solc/CommandLineParser.cpp

```diff
@@ -649,6 +650,16 @@ General Information)").c_str(),
 	;
 	desc.add(alternativeInputModes);
 
+	po::options_description lspModeOptions("LSP Mode Options");
+	lspModeOptions.add_options()
+		(
+			g_strLSPTrace.c_str(),
+			po::value<string>()->value_name(""),
+			"Enables trace I/O logging to a given file."
```

Comment:
Other options are described in the imperative:
```suggestion
			"Enable trace I/O logging to a given file."
```

Reply:
Also, I think this might be a bit clearer to users:

```suggestion
			"Log all JSON messages exchanged with an LSP client to a file."
```